src/features/unified-share-modal/HeaderTitle.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import Classification from '../classification';
+import messages from './messages';
+import type { item as ItemType } from './flowTypes';
+
+type Props = {
+    classification: ClassificationInfo,
+    isEmailLinkSectionExpanded: boolean,
+    item: ItemType,
+    showCollaboratorList: boolean,
+};
+
+function getTitle(isEmailLinkSectionExpanded, showCollaboratorList, item) {
+    const { name } = item;
+    let title;
+
+    if (isEmailLinkSectionExpanded) {
+        title = (
+            <FormattedMessage
+                {...messages.emailModalTitle}
+                values={{
+                    itemName: name,
+                }}
+            />
+        );
+    } else if (showCollaboratorList) {
+        title = (
+            <FormattedMessage
+                {...messages.collaboratorListTitle}
+                values={{
+                    itemName: name,
+                }}
+            />
+        );
+    } else {
+        title = (
+            <FormattedMessage
+                {...messages.modalTitle}
+                values={{
+                    itemName: name,
+                }}
+            />
+        );
+    }
+
+    return title;
+}
+
+const HeaderTitle = ({ classification, isEmailLinkSectionExpanded, showCollaboratorList, item }: Props) => {
+    const title = getTitle(isEmailLinkSectionExpanded, showCollaboratorList, item);
+    const { advisoryMessage, name } = classification;
+
+    return (
+        <span className="modal-title-label">
```

Comment:
Follow the SUIT css naming convention for this file, so `<span className="bdl-HeaderTitle">` and the classification section below `className="bdl-HeaderTitle-classification"`

Though thinking about it, bdl-HeaderTitle seems to give no information about it being part of usm.

Reply:
can we have the usm as midifierName `bdl-HeaderTitle--usm` and `bdl-HeaderTitle-classification--usm` ?

Reply:
Thats generally not the use case for modifiers. Modifiers are more like `button` to `button--plain`, that just modify the styles a bit. Options for this would be

1. usm-HeaderTitle, usm-HeaderTitle-classification
2. bdl-UnifiedSharedModal-headerTitle, bdl-UnifiedSharedModal-headerTitleClassification
3. bdl-UnifiedSharedModalHeaderTitle, bdl-UnifiedSharedModalHeaderTitle-classification

For [3] also rename the file name to match.